### PR TITLE
Quarkus testing

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

**Comment by zakkak** [comment]
It currently uses labsJDK. Should we change it to use EA OpenJDK 11 with static libs or wait for them to become GA?

**Comment by galderz** [comment]
I'd suggest EA OpenJDK 11. Can you target B4? That's what I'm targeting on my side.

**Comment by jerboaa** [comment]
> I'd suggest EA OpenJDK 11. Can you target B4? That's what I'm targeting on my side.

+1

We should use OpenJDK 11.0.8 right from the start. Grab it from here:
https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/tag/jdk-11.0.8%2B4

**Comment by zakkak** [comment]
Done

**Comment by jerboaa** [comment]
Thanks. This seems fine. I cannot really review the yaml as I've got no experience with them, sorry.

**Comment by zakkak** [comment]
np, if @galderz is also OK we can merge this.

File: .github/workflows/quarkus.yml [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+        fetch-depth: 1
+        path: mandrel
+    - uses: actions/checkout@v1
+      with:
+        repository: graalvm/mx.git
+        ref: master
+        path: mx
+    - name: Get latest quarkus release
+      run: |
+        curl --output quarkus.tgz -sL $(curl -sL https://api.github.com/repos/quarkusio/quarkus/releases/latest | jq -r .tarball_url)
```
Comment (galderz) [inline .github/workflows/quarkus.yml:31]: Hmmm, I wonder if we should test against master instead. Master tested with master might be more useful.

**Comment by galderz** [comment]
@zakkak Are there any runs of this GH action available? Or I guess we have to wait for this to be pushed? Also, what do we want to do with existing Travis CI? Remove it? Keep it? Is it of any use to us?

File: .github/workflows/quarkus.yml (hunk 1 above)
Comment (zakkak) [reply .github/workflows/quarkus.yml:31]: I agree, I would actually suggest testing both the latest release *and* master.

**Comment by zakkak** [comment]
> @zakkak Are there any runs of this GH action available? 

https://github.com/zakkak/mandrel/actions/runs/114921785

> Also, what do we want to do with existing Travis CI? Remove it? Keep it? Is it of any use to us?

I suggest we keep it, we don't want to break anything working upstream. The only issue is that upstream does not ensure Travis is always green (e.g. on 20.1 all JDK8 tests fail, plus 2 JDK11 tests on AArch64) so we have to make sure we have the same failures (which is harder to check).

**Comment by zakkak** [comment]
Latest CI results https://github.com/zakkak/mandrel/actions/runs/118054517
Note that some of the failures are due to https://quay.io/ being down.

**Pull request merged**